pkg/controllers/metrics/pod/controller.go

```diff
@@ -100,19 +101,40 @@ var (
 	PodUnstartedTimeSeconds = opmetrics.NewPrometheusGauge(
 		crmetrics.Registry,
 		prometheus.GaugeOpts{
-			Namespace: "karpenter",
+			Namespace: metrics.Namespace,
 			Subsystem: metrics.PodSubsystem,
 			Name:      "unstarted_time_seconds",
 			Help:      "The time from pod creation until the pod is running.",
 		},
 		[]string{podName, podNamespace},
 	)
+	PodScheduledDurationSeconds = opmetrics.NewPrometheusGauge(
+		crmetrics.Registry,
+		prometheus.GaugeOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.PodSubsystem,
+			Name:      "scheduled_seconds",
```

Comment:
```suggestion
			Name:      "unscheduled_time_seconds",
```

Are we tracking the amount of time that it took a pod to schedule as a gauge or are we tracking how long the pod is in a state where it hasn't scheduled? I think the latter is more valuable -- the first one you should be able to get from the histogram metric

Reply:
Ah good call